app/api/events_role_permission.py

```diff
@@ -0,0 +1,38 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, \
+    ResourceRelationship
+
+from app.api.bootstrap import api
+from app.api.schema.event_role_permissions import EventsRolePermissionSchema
+from app.models import db
+from app.models.permission import Permission
+
+
+class EventsRolePermissionList(ResourceList):
+    """
+    List Events Role Permission
+    """
+    decorators = (api.has_permission('is_admin', methods="GET"),)
+    schema = EventsRolePermissionSchema
+    data_layer = {'session': db.session,
+                  'model': Permission}
```

Comment:
Is POST request not allowed on this endpoint ? 
If not allowed then add the methods field:
``` Python
methods = ['GET']
```

If allowed, then add it to the methods in the decorator so that we check for permissions in the POST request also.